<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CModelForm</class>
 <widget class="QDialog" name="CModelForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模型库</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableWidget" name="listModel">
     <column>
      <property name="text">
       <string>模型ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>模型名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>中文名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>建立时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>厂站名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备名</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="buttonParam">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>模型参数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAdd">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>添加模型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonDel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>删除模型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonModify">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>修改模型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonModelInto">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>导入模型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonModelConfig">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>打开模型配置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonStnDev">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>选择厂站设备</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonRefresh">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonExit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonParam</tabstop>
  <tabstop>buttonAdd</tabstop>
  <tabstop>buttonDel</tabstop>
  <tabstop>buttonModify</tabstop>
  <tabstop>buttonRefresh</tabstop>
  <tabstop>buttonExit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonExit</sender>
   <signal>clicked()</signal>
   <receiver>CModelForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>695</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
